ansible_deploy/command_line.py

```diff
@@ -56,10 +51,14 @@ def set_logging(log_dir: str, name: str, timestamp: str):
     console_handler.setLevel(logging.DEBUG)
     logger.addHandler(console_handler)
 
-    file_handler = logging.FileHandler(log_path)
-    file_handler.setFormatter(formatter)
-    file_handler.setLevel(logging.DEBUG)
-    logger.addHandler(file_handler)
+    if log_dir:
+        if not os.path.exists(log_dir):
+            os.makedirs(log_dir)
```

Comment:
os.makedirs has a parameter: "exist_ok=bool" but I'm not sure what goes on low-level. In other case try/except would prolly be better.

Reply:
it sounds better since the old approach is vulnerable to TOCTOU race